### PR TITLE
Feature/ted 46

### PR DESCRIPTION
**Pull request by CaptainOfHacks**
extract an RDF fragment corresponding to a CET instance

### CODE REVIEW

File: ted_sws/master_data_registry/model/rdf_fragment.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Author: Stratulat Ștefan
+# Email: stefan.stratulat1997@gmail.com
+from typing import List, Tuple
+
+from rdflib.term import URIRef
+
+from ted_sws.core.model import PropertyBaseModel
+
+
+class RDFFragment(PropertyBaseModel):
```
Comment (costezki) [inline ted_sws/master_data_registry/model/rdf_fragment.py:14]: rdflib.Graph is already an RDFFragment. 

Please do not create this class unless you need it for other purposes, which may be. But NOT for storing a list of triplets.

File: ted_sws/master_data_registry/services/rdf_fragment_processor.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+from string import Template
+from typing import List
+
+from rdflib import Graph
+
+from ted_sws.master_data_registry.model.rdf_fragment import RDFFragment
+from ted_sws.master_data_registry.resources import RDF_FRAGMENT_BY_URI_SPARQL_QUERY_TEMPLATE_PATH, \
+    TRIPLES_BY_CET_URI_SPARQL_QUERY_TEMPLATE_PATH
+
+DEFAULT_RDF_FILE_FORMAT = "n3"
```
Comment (costezki) [inline ted_sws/master_data_registry/services/rdf_fragment_processor.py:21]: why `n3` and not Turtle?

File: ted_sws/master_data_registry/services/rdf_fragment_processor.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+
+def get_rdf_fragment_by_cet_uri_from_string(rdf_file_content: str, cet_uri: str,
```
Comment (costezki) [inline ted_sws/master_data_registry/services/rdf_fragment_processor.py:24]: the logic in this function shall be implemented as a new class in the data_manager sparql_endpoint.py module.

the class shall be 

```
class SPARQLFileEndpoint(TripleStoreEndpointABC):
    ...
```
if necessary you may also create something to parse and query an arbitrary string, but I don't think we need it. 

```
class SPARQLStringEndpoint(TripleStoreEndpointABC):
    ...
``` 


image

File: tests/fakes/fake_triple_store.py [hunk 4]
```diff
@@ -1,9 +1,15 @@
+from typing import List, Tuple
+
 import pandas as pd
+from rdflib import URIRef
 
 from ted_sws.data_manager.adapters.sparql_endpoint import TripleStoreEndpointABC
 
 
 class FakeTripleStoreEndpoint(TripleStoreEndpointABC):
+    def fetch_rdf(self) -> List[Tuple[URIRef, URIRef, URIRef]]:
```
Comment (costezki) [inline tests/fakes/fake_triple_store.py:10]: nonononono, do nto operate on list os triplets pls. 
user `rdflib.Graph()`

**Review by costezki — CHANGES_REQUESTED** [review]
good start, we need a bit of refactoring.

File: ted_sws/master_data_registry/services/rdf_fragment_processor.py (hunk 2 above)
Comment (CaptainOfHacks) [reply ted_sws/master_data_registry/services/rdf_fragment_processor.py:21]: n3 is used also for Turle, based on rdflib documentation

**Review by costezki — APPROVED** [review]
Looks neat!

**Pull request merged**